src/validators/mod.rs

```diff
@@ -441,9 +442,12 @@ pub fn build_validator<'a>(
 /// but that would confuse it with context as per pydantic/pydantic#1549
 #[derive(Debug, Default)]
 pub struct Extra<'a> {
-    /// This is used as the `data` kwargs to validator functions, it also represents the current model
-    /// data when validating assignment
+    /// Represents the fields of the model we are currently validating
+    /// If there is no model this will be None
     pub data: Option<&'a PyDict>,
+    /// Represents the fields of the model we are currently validating
+    /// If there is no model this will be None
+    pub field_name: Option<&'a str>,
     /// The field being assigned to when validating assignment
     pub field: Option<&'a str>,
```

Comment:
we should probably rename this to `assignment_field` or similar, could do later.

Reply:
surely not.

`assignee_field`?

Reply:
Works for me